xml/System.Windows/ThemeModeConverter.xml

```diff
@@ -88,10 +90,10 @@
         <Parameter Name="destinationType" Type="System.Type" />
       </Parameters>
       <Docs>
-        <param name="typeDescriptorContext">To be added.</param>
-        <param name="destinationType">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="typeDescriptorContext">The ITypeDescriptorContext for this call.</param>
+        <param name="destinationType">The Type being queried for support.</param>
+        <summary>CanConvertTo - Returns whether or not this class can convert to a given type.</summary>
+        <returns>bool - <see langword="true" /> if this converter can convert to the provided type, <see langword="false" /> if not.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true" /> if this converter can convert to the provided type; otherwise, <see langword="false" />.</returns>
```